test/browser.js

```diff
@@ -85,11 +85,13 @@ test('throws TimeoutError even though it does not support AbortController', with
 	await page.addScriptTag({path: './test/helpers/disable-abort-controller.js'});
 	await page.addScriptTag({path: './umd.js'});
 
+	let request;
 	const error = await page.evaluate(url => {
-		const request = window.ky(`${url}/slow`, {timeout: 500}).text();
+		request = window.ky(`${url}/slow`, {timeout: 500}).text();
 		return request.catch(error_ => error_.toString());
 	}, server.url);
 	t.is(error, 'TimeoutError: Request timed out');
+	t.is(error.request, request);
```

Comment:
Haven't checked yet, but would be nice if `ava` caught this somehow...

Reply:
Could probably make an XO linter rule that checks the `page.evaluate()` callback.

Reply:
Bet you could, but I was more thinking of the `t.is(undefined, undefined)` case.

Reply:
Yeah it would be nice but as you alluded to, I doubt it is feasible. While rare, I have occasionally wrote an assertion like that on purpose to check that an object has been modified in a certain way, for example. So I think the linter is more realistic.

Reply:
Just want to make sure: have you seen my response to the other comment?

Simply asking since you reply to this thread fairly quick. Just want to move this PR forward.

Thank you 🙏